pkg/relayer/miner/miner.go

```diff
@@ -102,7 +102,7 @@ func (mnr *miner) mapMineRelay(
 	relayHash := mnr.hash(relayBz)
 
 	// The relay IS NOT volume / reward applicable
-	if !protocol.BytesDifficultyGreaterThan(relayHash, defaultRelayDifficulty) {
+	if protocol.MustCountDifficultyBits(relayHash) >= defaultRelayDifficultyBits {
```

Comment:
Should this be `<` given that this is the `NOT volume applicable` codepath?

Reply:
Nice catch, thanks! :facepalm: I guess it's been another long week. :sweat_smile:

Reply:
FWIW, the tests added in #191 would've caught this. :mechanical_arm:

Reply:
I was going to just having a TODO to add tests for this regression but you're one step ahead!